providers/hns/hns_roce_u.h

```diff
@@ -113,6 +113,7 @@ struct hns_roce_buf {
 };
 
 #define BIT_CNT_PER_BYTE       8
+#define BIT_CNT_PER_LONG       (BIT_CNT_PER_BYTE * sizeof(uint64_t))
```

Comment:
It isn't really a long. BIT_CNT_PER_U64? Kind of a silly thing to write. Why not just write 64?